cfme/configure/settings.py

```diff
@@ -349,12 +349,17 @@ def __init__(self, appliance=None):
         Navigatable.__init__(self, appliance=appliance)
 
     @classmethod
-    def set_default_view(cls, button_group_name, default):
+    def save(cls):
```

Comment:
Well, Note that I need to set different default for each object, There is a problem with that since I'll also have to change the 'default' arg to 'defaults' as well, this can cause serious complexity right?
I can do it if you want.